API/Controllers/WizardController.cs

```diff
@@ -266,7 +266,7 @@ public class WizardController : ControllerBase
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
         public async Task<IActionResult> GetProjectByGuidFromExternalDataSource([FromQuery] string dataSourceGuid,
             [FromQuery] string token,
-            int projectId,
+            string projectId,
```

Comment:
Why did you change this to a string?

Reply:
This method will work with external API calls (indirectly). Because some external data sources will get added later, we don't know the specific specifications of that API. Some API's might use integers as id's for their projects, but some other API's might use guids, or even other data types. 

So, therefore I decided to use the string. I can use guids in a string, but also integers. Later in the process, every adaptee can parse this string in the way their API needs to id delivered.

Good question and thanks for the reminder :wink: